torchvision/datasets/utils.py

```diff
@@ -221,7 +201,7 @@ def download_file_from_google_drive(
         filename (str, optional): Name to save the file under. If None, use the id of the file.
         md5 (str, optional): MD5 checksum of the download. If None, do not check
     """
-    # Based on https://stackoverflow.com/questions/38511444/python-download-files-from-google-drive-using-url
+    import gdown
```

Comment:
Maybe we should try/except (in a helper?) and add a descriptive error message linking to `https://github.com/wkentaro/gdown` or directly indicating to install with `pip install gdown`